### DISCUSSION TITLE
Styling of jointjs paperscroller scrollbar

### DISCUSSION THREAD

**Discussion by ROYopedia** [clientIO/joint#2203] [Q&A]
### Introduction

I am trying to custom style scrollbar appeared with paper scroller. Below is one simple example.
![image](https://github.com/clientIO/joint/assets/22453886/d9c4aa91-723e-45e9-a16b-d6e2bbea586f)

I have tried with css using -webkit-scrollbar and others but it seems styling is not applying.
Could you please give some information on this? 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by jamesgeorgewilliams** [comment]
You can use the `::-webkit-scrollbar` set of CSS pseudo-elements.
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

Although, it is considered non-standard to use.
https://caniuse.com/?search=%3A%3A-webkit-scrollbar

<img width="1196" alt="Screenshot 2023-06-06 at 9 49 39" src="https://github.com/clientIO/joint/assets/42288565/ab6f8419-d8d1-4e4d-a103-4b7b5da51334">

**Reply by ROYopedia** [reply]
Same thing. I am worried about the nonstandard thing.

**Reply by kumilingus** [reply]
This is a general question not directly related to JointJS. The paper scroller is just a `div` element with `overflow: scroll`. To style the scrollbars please search the solution on the web.

For instance, to style the scrollbar in _Chrome_ or _Safari_ follow this: https://css-tricks.com/almanac/properties/s/scrollbar/
For _Firefox_ use this:https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_scrollbars_styling
